subsys/fb/Kconfig

```diff
@@ -31,6 +50,8 @@ if CHARACTER_FRAMEBUFFER_SHELL
 	string
 	default "SSD1673" if SSD1673
 	default "SSD1306" if SSD1306
+	default "TBD"
+	depends on CHARACTER_FRAMEBUFFER_SHELL && CHARACTER_FRAMEBUFFER
```

Comment:
No need for `depends on` as the config is already in identical if statements

Reply:
ok